ui/app/styles/app.scss

```diff
@@ -4,9 +4,7 @@
 @use 'pds/utils/layout' as Layout;
 @use 'pds/tokens/font';
 @use 'pds/core/typography/config' as Typography;
-@use 'utilities/animations';
-
-@use 'utilities/focus-ring';
+@use 'utilities/index';
```

Comment:
Probably nitpicky: Do you need to specify index here? I _think_ you can do @use 'utilities'; and it will use the index file.  

Not sure if we have a preferred way of doing this type of import.

Reply:
thanks. probably showing my age, but I've worked with older versions of Sass that didn't support the implicit index, so I didn't even try doing that 🧓

Reply:
And here I was thinking that maybe they'd removed the implicit index since I last used it 😅